yellowbrick/classifier/classification_report.py

```diff
@@ -122,17 +136,17 @@ def draw(self):
         Renders the classification report across each axis.
         """
         # Create display grid
-        cr_display = np.zeros((len(self.classes_), 4))
+        cr_display = np.zeros((len(self.classes_), len(self.measurements)))
```

Comment:
Awesome, thank you! This definitely will make things a bit more dynamic if we want to allow users to display different metrics.